kubernetes/resource_kubernetes_job_test.go

```diff
@@ -16,6 +17,44 @@ func ttlAfterDisabled() (bool, string) {
 	return os.Getenv("FEATURE_GATE_TTL_AFTER_FINISHED") != "enabled", "TTLAfterFinished is not enabled"
 }
 
+func TestAccKubernetesJob_wait_for_completion(t *testing.T) {
```

Comment:
@ichekrygin I added a separate test just for this functionality to make it a bit easier to understand what is being tested.